src/api/todos.js

```diff
@@ -0,0 +1,76 @@
+// Todos table functions 
+import axios from './axios';
+
+// RESTful API endpoint for todo objects
+const endpoint = '/todos';
+
+/**
+ * Get all rows in the todos table.
+ * @returns array of todos 
+ */
+async function getTodos() {
+  const { data } = await axios.get(endpoint);
+  return data;
+}
+
+/**
+ * Insert a new todo in the todos table
+ * @param {*} todo new todo to insert 
+ * @returns true if success, false if not
+ */
+async function postTodo(todo) {
+  const { status } = await axios.post(endpoint, todo);
```

Comment:
Please conform with RESTful HTTP semantics. PostTodo API call creates a new resource to the server. According to this [reference](https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design#conform-to-http-semantics) on POST requests that creates a new resource to server:

> If a POST method creates a new resource, it returns HTTP status code 201 (Created). The URI of the new resource is included in the Location header of the response. The response body contains a representation of the resource.

I will adjust the API endpoints as such.

Similar with my comment on the get request, please return the full api response (header + body) on the caller (from component). Also please check with the 201 status code not on 200. We might need also need response body on some features on the future.

Also please do the error handling on the caller, I think moving forward when can have a global error handler for all these requests (as adding try-catch block for every api service caller is not DRY).